README.md

```diff
@@ -21,6 +21,11 @@
 
 ✔ Support extracting with a subdir.
 
+## Comparison
+
+- Does not depend on local git to be installed ([read more](https://github.com/unjs/giget/issues/3#issuecomment-1240667190))
```

Comment:
Would be better to make it a sentence directly in readme and also in [related projects](https://github.com/unjs/giget#related-projects) section. We shall move it above development

Reply:
Does that mean we explain the tradeoffs and diffs in "related projects" section?

Reply:
Yes i think that is more relavant section.

Reply:
I've pushed some changes :)